### PR TITLE
WEBEU3 - JavaScript-I - Ifiok-Udoidiok

### PR DESCRIPTION
**Pull request by ifiokudoidiok**

### CODE REVIEW

File: assignments/arrays.js [hunk 1]
```diff
@@ -75,30 +75,77 @@ let inventory = [
...
+//       if (inventory[33].id === 33) {
+//           return inventory[i];
+//       }
+//   }
+// }
+
+// console.log(inventory[32].car_year);
+
+// var resultObject = search("string 1", array);
+console.log(`Car 33 is a ${inventory[32].car_year} ${inventory[32].car_make} ${inventory[32].car_model}`);
```
Comment (josenriagu) [inline assignments/arrays.js:90]: what happens when items before this car get deleted?

definitely inventory[32] will no longer be the car with id 33. You can try a for loop and look for the id instead.

File: assignments/function-conversion.js [hunk 2]
```diff
@@ -5,20 +5,31 @@
...
 // add(1,2);
 
+let add = (param1, param2) => (param1 + param2);
+add(1,2);
+
 // let subtract = function (param1, param2) {
 //   return param1 - param2;
 // };
 // subtract(1,2);
+let subtract = (param1, param2) => (param1 - param2);
```
Comment (josenriagu) [inline assignments/function-conversion.js:31]: Good job

File: assignments/objects.js [hunk 3]
```diff
@@ -49,16 +94,41 @@ const example = {
...
 // Have the parent speak
+parent.speak = () => `Hello, my name is ${parent.name}`;
+console.log(parent.speak());
 
 // Have the child speak
+parent.child.speak = () => `Hello, my name is ${parent.child.name}`;
+console.log(parent.child.speak());
 
 // Have the grandchild speak
+parent.child.grandchild.speak = () => `Hello, my name is ${parent.child.grandchild.name}`;
```
Comment (josenriagu) [inline assignments/objects.js:133]: beautiful!

**Review by josenriagu — APPROVED** [review]
Good job!

Keep it up

File: assignments/arrays.js (hunk 1 above)
Comment (ifiokudoidiok) [reply assignments/arrays.js:90]: Oh My mistake.
There is a for-loop in place in the search function. I just did not put it into good use. I will correct that.
Thanks!

**Pull request merged**